tests/testthat/test_types.R

```diff
@@ -23,8 +23,10 @@ test_that('types.R: showTypes()', {
     expect_equal(length(showTypes_DGEobj), 50)
 
     warning('function does not appear to be working')
-    # showTypes_notPretty_DGEobj <- showTypes(DGEobj, pretty = FALSE)
-    # expect_equal(length(showTypes_notPretty_DGEobj), 50)
+    showTypes_notPretty_DGEobj <- DGEobj::showTypes(DGEobj, pretty = FALSE)
```

Comment:
remove DGEobj:: unless necessary in testing files - we should not need use of double colons in the tests